src/core.ts

```diff
@@ -34,6 +35,7 @@ smoothscroll.polyfill();
     StachePageHeaderModule,
     StacheLayoutModule,
     StacheWrapperModule
-  ]
+  ],
+  providers: [ WindowRef ]
```

Comment:
Hmm. The window reference is already used in 3 modules. I think having a global window reference makes sense.